Sprint-2/1-key-errors/0.js

```diff
@@ -1,13 +1,19 @@
 // Predict and explain first...
 //  =============> write your prediction here
-
+// it gives us an error.
 // call the function capitalise with a string input
 // interpret the error message and figure out why an error is occurring
 
-function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
-  return str;
-}
+// function capitalise(str) {
```

Comment:
Let's remove all commented code/console logs/any scratch work that is not relevant to the pull request to keep it clean.